js/deer-config.js

```diff
@@ -886,6 +924,17 @@ export default {
                         })
                         containingListElem.setAttribute("ng-list-loaded", "true")
                         deerUtils.broadcast(undefined, "ng-list-loaded", containingListElem, {})
+						const searchSubmit = document.getElementById('search-submit')
```

Comment:
If this is just copy-paste from line 613, it should be pulled out into a function and reused.